docker/dyno/tests/conftest.py

```diff
@@ -19,7 +19,10 @@
 """
 Tests for Dyno control plane
 """
-import toxiproxy
+try:
+    import toxiproxy
```

Comment:
This just cleans up some errors that can happen with pytest during test discovery.